cmd/openfga/openfga.go

```diff
@@ -203,3 +207,28 @@ func buildService(logger logger.Logger) (*service, error) {
 		authenticator: authenticator,
 	}, nil
 }
+
+func buildLogger() (*logger.ZapLogger, error) {
+	var err error
+	var openfgaLogger *logger.ZapLogger
```

Comment:
Should the initialization default to the default value of the config and then branch into the non default?